src/ngraph/builder/quantization/quantized_linear_convolution.cpp

```diff
@@ -56,6 +61,48 @@ namespace ngraph
                                                              requantization_scale);
             }
 
+            shared_ptr<Node> QuantizedLinearConvolution(shared_ptr<Node> input,
```

Comment:
QuantizedLinearConvolutionFallback? or Add comments on why we are falling back to fp32.